superagi/resource_manager/resource_manager.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from llama_index import SimpleDirectoryReader
+
+from superagi.config.config import get_config
+from superagi.helper.resource_helper import ResourceHelper
+from llama_index.schema import Document
+
+from superagi.lib.logger import logger
+from superagi.types.vector_store_types import VectorStoreType
+from llama_index.indices.response import ResponseMode
+
+
+class ResourceManager:
+    @classmethod
+    async def create_llama_document(cls, file_path: str = None, file_object=None):
```

Comment:
If only one param exists then this method does not make sense. Create different method for s3 and file.